### DISCUSSION TITLE
Javascript Example

### DISCUSSION THREAD

**Discussion by sugoidesune** [TanStack/table#5323] [Q&A]
Is there a working example somewhere implemented with javascript only, without typescript?
I don't really understand what the columnHelper does taking the types etc.

**Comment by shuvro111** [comment]
@sugoidesune This discussion (#4096)is also about this issue. I think there is no official example with Javascript till now.
However, I found this [example](https://github.com/OneLightWebDev/react-table-tutorial) with javascript and this [YouTube video](https://www.youtube.com/watch?v=CjqG277Hmgg) explains it. You can also take a look at this [codesandbox](https://codesandbox.io/p/github/shuvro111/react-table-tutorial/main?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clt2vp7e300063b6hdewxlrox%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clt2vp7e300023b6h14kh17by%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clt2vp7e300043b6hpyf2n9oh%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clt2vp7e300053b6h99di7zqi%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clt2vp7e300023b6h14kh17by%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clt2vp7e200013b6hze2pon4t%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252FREADME.md%2522%257D%255D%252C%2522id%2522%253A%2522clt2vp7e300023b6h14kh17by%2522%252C%2522activeTabId%2522%253A%2522clt2vp7e200013b6hze2pon4t%2522%257D%252C%2522clt2vp7e300053b6h99di7zqi%2522%253A%257B%2522id%2522%253A%2522clt2vp7e300053b6h99di7zqi%2522%252C%2522activeTabId%2522%253A%2522clt2vrczp00he3b6hd9kx39mz%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A5173%252C%2522id%2522%253A%2522clt2vrczp00he3b6hd9kx39mz%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%257D%252C%2522clt2vp7e300043b6hpyf2n9oh%2522%253A%257B%2522id%2522%253A%2522clt2vp7e300043b6hpyf2n9oh%2522%252C%2522activeTabId%2522%253A%2522clt2vpa7i00453b6hz6w2075m%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clt2vp7e300033b6h4g3dyb6f%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522clt2vp96x0011dlijabwceomh%2522%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522id%2522%253A%2522clt2vpa7i00453b6hz6w2075m%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%252C%257B%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522CSB_RUN_OUTSIDE_CONTAINER%253D1%2520devcontainer%2520templates%2520apply%2520--template-id%2520%255C%2522ghcr.io%252Fdevcontainers%252Ftemplates%252Fjavascript-node%255C%2522%2520--template-args%2520%27%257B%257D%27%2520--features%2520%27%255B%255D%27%2522%252C%2522id%2522%253A%2522clt2vprcf005o3b6hqc3vokea%2522%252C%2522mode%2522%253A%2522permanent%2522%257D%255D%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D) link to see it in action.

**Reply by sugoidesune** [reply]
I read in one reply somewhere that it works if you just simply delete all typescript code with no extra steps.  I think I managed to get it to work like that. Ultimately I wish there was a UI also available not just the logic.

**Reply by shuvro111** [reply]
A CLI tool or command would be awesome in that case.
